packages/common/src/utils/urlFormatting.ts

```diff
@@ -125,6 +132,17 @@ export const contentListURL = (id: string, title?: string) => {
   return createPath(PATH_CONTENT_LIST, { id, title: title ? slugify(title) : undefined });
 };
 
+export const determinePath = ({ type, contentId }: { type: AppMenuType | undefined; contentId: string }) => {
+  switch (type) {
+    case APP_CONTENT_TYPE.content_list:
+      return contentListURL(contentId);
+    case APP_CONTENT_TYPE.media:
+      return singleMediaURL(contentId);
+    default:
```

Comment:
We should probably check playlist explicitly, as it is still possible we get something extra in the future.